src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task {
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
+}
+
```

Comment:
overall, the code is very well written and follow coding standard. for the most part it is very easily readable, but the processAndExecute function is a bit too long, and could be broken up into smaller functions